src/r_main.c

```diff
@@ -818,7 +823,9 @@ void R_RenderPlayerView (player_t* player)
 
   // The head node is the last node output.
   R_RenderBSPNode (numnodes-1);
-    
+
```

Comment:
What does this do if no voxels are available?

Reply:
> What does this do if no voxels are available?

What I mean is, I am afraid that this affects regular sprite rendering.

Reply:
If no model exists for the given sprite? Then `VX_ProjectVoxel()` exits, and `R_ProjectSprite()` continues to draw the standard sprite.

`VX_NearbySprites()` handles the special case of models disappearing in small sectors that have gone out of view. I haven't tested it, I'm not sure how often it's needed.

> What I mean is, I am afraid that this affects regular sprite rendering.

As far as I know, it doesn't affect it in any way. The code looks good.

Reply:
Thank you!